src/Controller/CRUDController.php

```diff
@@ -1142,12 +1130,28 @@ protected function redirectTo($object)
         }
 
         if (!$url) {
-            $url = $this->admin->generateUrl('list');
+            return $this->redirectToList();
         }
 
         return new RedirectResponse($url);
     }
 
+    /**
+     * Redirects the user to the list view.
+     *
+     * @return RedirectResponse
+     */
+    final protected function redirectToList()
+    {
+        $parameter = [];
```

Comment:
Did you really mean to make this variable singular?

Reply:
Fixed